### PR TITLE
Selenium: Update javadocs to actual state in the "CommandsEditor" class

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Update javadocs to actual state in the "CommandsEditor" class

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/9238

### CODE REVIEW

**Review by artaleks9 — APPROVED** [review]
It is good for me

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java [hunk 1]
```diff
@@ -324,85 +324,152 @@ public void waitTabIsNotPresent(String nameOfFile) {
...
+  public void waitTabCommandWithUnsavedStatus(String commandName) {
     seleniumWebDriverHelper.waitVisibility(
-        By.xpath(format(ACTIVE_TAB_UNSAVED_FILE_NAME, nameCommand)), ELEMENT_TIMEOUT_SEC);
+        By.xpath(format(ACTIVE_TAB_UNSAVED_FILE_NAME, commandName)), ELEMENT_TIMEOUT_SEC);
   }
 
-  public void typeTextIntoNameCommandField(String nameCommand) {
-    seleniumWebDriverHelper.setValue(nameCommandField, nameCommand);
+  /**
+   * Types specified {@code commandName} to the "Command Name" field, and waits until typed text is
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java]: I would mention updating of specified command immediately without additional notes which requires special attention and additional time: `Sets name of command to {@code commandName} and wait until it appears.`

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java [hunk 2]
```diff
@@ -324,85 +324,152 @@ public void waitTabIsNotPresent(String nameOfFile) {
...
   public void setFocusIntoTypeCommandsEditor(String commandsEditorType) {
     seleniumWebDriverHelper.waitAndClick(
         By.xpath(commandsEditorType), REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
     editor.waitActive();
   }
 
+  /**
+   * Waits until specified {@code expectedText} is displayed in the popup with macro description.
+   *
+   * <p>Note! Popup with description opens after single click on item in the macro autocomplete
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java]: looks misleading - like method itself click on item, but it doesn't.
I would remove it so as there could be another way to open macro form.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java [hunk 3]
```diff
@@ -324,85 +324,152 @@ public void waitTabIsNotPresent(String nameOfFile) {
...
+   *
+   * @param macroLinkType type of the "Macros" link, defined in {@link CommandsMacrosLinkType}
+   */
+  public void selectMacroLinkInCommandsEditor(String macroLinkType) {
+    seleniumWebDriverHelper.waitAndClick(By.xpath(macroLinkType), REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
     waitCommandsMacrosIsOpen();
   }
 
+  /** Waits until "Command Macros" form is opened. */
   public void waitCommandsMacrosIsOpen() {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java]: `waitCommandMacrosIsOpen` name looks more relevant here

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java [hunk 4]
```diff
@@ -324,85 +324,152 @@ public void waitTabIsNotPresent(String nameOfFile) {
...
     seleniumWebDriverHelper.waitVisibility(
         By.xpath(COMMAND_MACROS_FORM), REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
   }
 
+  /** Waits until "Command Macros" form is closed. */
   public void waitCommandsMacrosIsClosed() {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java]: `waitCommandMacrosIsClosed` name looks more relevant here

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java [hunk 5]
```diff
@@ -324,85 +324,152 @@ public void waitTabIsNotPresent(String nameOfFile) {
...
     seleniumWebDriverHelper.waitInvisibility(
         By.xpath(COMMAND_MACROS_FORM), REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
   }
 
+  /**
+   * Types and waits specified {@code text} to the "Search" field in the "Command Macros" form.
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java]: This JavaDoc will look simpler without additional note, IMHO. Note can be emphasized in first sentence, for example:
`Sets specified {@code text} to the "Search" field in the "Command Macros" form and waits until it appears.`

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java [hunk 6]
```diff
@@ -411,27 +478,52 @@ public void waitTextIntoMacrosContainer(final String expectedText) {
                 webDriver -> getAllVisibleTextFromMacrosContainer().contains(expectedText));
   }
 
+  /**
+   * Gets all visible text from proposal area of the "Command Macros" form.
+   *
+   * @return visible text from proposal area
+   */
   public String getAllVisibleTextFromMacrosContainer() {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java]: It seems method name could be rephrased to `getVisibleTextFromMacrosContainer` without loss of meaning.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java [hunk 7]
```diff
@@ -411,27 +478,52 @@ public void waitTextIntoMacrosContainer(final String expectedText) {
...
     waitCommandsMacrosIsOpen();
     return macrosContainer.getText();
   }
 
+  /**
+   * Performs entering the specified {@code item} by single clicking and sending of the "Enter" key.
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java]: JavaDoc sounds over complicated as for me. how about simpler one: `Enters macro command {@code item} by pressing Enter key`

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java [hunk 8]
```diff
@@ -411,27 +478,52 @@ public void waitTextIntoMacrosContainer(final String expectedText) {
...
+   */
   public void enterMacroCommandByEnter(String item) {
     selectMacroCommand(item);
     seleniumWebDriverHelper.sendKeys(ENTER.toString());
   }
 
+  /**
+   * Performs entering the specified {@code item} by single clicking and performing double clicking.
+   *
+   * @param item macro body in format "${current.project.path}" without description
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java]: JavaDoc sounds over complicated as for me. how about simpler one: `Enters macro command {@code item} by pressing Enter key`

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**